flutter_todo/android/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
It required the latest version of Kotlin 1.6.21 when I run it on Android on my environment.